commons-numbers-primes/src/main/java/org/apache/commons/numbers/primes/SmallPrimes.java

```diff
@@ -108,11 +108,11 @@ equivalence classes will have to hold (2-1)*(3-1)*(5-1)*(7-1)*(11-1) = 480
         of all integers.
          */
         final Set<Integer> primeNumbers = new HashSet<>();
-        primeNumbers.add(Integer.valueOf(2));
-        primeNumbers.add(Integer.valueOf(3));
-        primeNumbers.add(Integer.valueOf(5));
-        primeNumbers.add(Integer.valueOf(7));
-        primeNumbers.add(Integer.valueOf(11));
+        primeNumbers.add(2);
```

Comment:
The current code makes the boxing explicit. Please avoid auto-boxing.

Reply:
Changed